### PR TITLE
Switch to wdl 1.0

### PR DESCRIPTION
**Pull request by ffinfo**

### CODE REVIEW

File: pipeline.wdl [hunk 1]
```diff
@@ -20,32 +22,37 @@
...
-            inputFiles = sampleConfigFiles
+            inputFiles = sampleConfigFiles,
+            outputPath = "samples.json"
     }
 
-    # Do the jobs that should be executed per sample.
-    # Modify sample.wdl to change what is happening per sample
-    scatter (sampleId in samplesConfigs.keys) {
-        call sampleWorkflow.sample as sample {
+     Root config = read_json(configFile.outputFile)
```
Comment (DavyCats) [inline pipeline.wdl]: The indentation here has an excess space.

File: readgroup.wdl [hunk 2]
```diff
@@ -21,28 +23,25 @@
...
     }
 
     call QC.QC as qc {
         input:
-            read1 = config.values.R1,
-            read2 = config.values.R2,
-            outputDir = outputDir + "qc"
+            read1 = readgroup.R1,
+            read2 = readgroup.R2,
+            outputDir = readgroupDir,
```
Comment (DavyCats) [inline readgroup.wdl:41]: Shouldn't the QC have its own output directory?

File: sample.wdl [hunk 3]
```diff
@@ -23,88 +25,83 @@ import "tasks/biopet.wdl" as biopet
...
+    Int seed = select_first([virusAssemblyInputs.downsampleSeed, 11])
+
+    if (defined(virusAssemblyInputs.fractionOrNumber)) {
+        call seqtk.Sample as subsampleRead1 {
+            input:
+                sequenceFile = combinedReads1,
+                fractionOrNumber = select_first([virusAssemblyInputs.fractionOrNumber]),
+                seed = seed,
+                outFilePath= sampleDir + "/subsampling/subsampledReads1.fq.gz", #Spades needs a proper extension or it will crash
+                zip=true
```
Comment (DavyCats) [inline sample.wdl]: `=` should be surrounded with spaces

File: sample.wdl [hunk 4]
```diff
@@ -23,88 +25,83 @@ import "tasks/biopet.wdl" as biopet
...
 
     # Call spades for the de-novo assembly of the virus.
-    call spades.spades {
+    call spades.Spades as spades {
         input:
-            read1=subsampleRead1.subsampledReads,
-            read2=subsampleRead2.subsampledReads,
-            outputDir=outputDir + "/spades"
+            read1 = select_first([subsampleRead1.subsampledReads, combinedReads1]),
+            read2 = if (defined(virusAssemblyInputs.fractionOrNumber)) then subsampleRead2.subsampledReads else combinedReads2,
```
Comment (DavyCats) [inline sample.wdl]: This line is rather long. Please but a newline before the `then` and `else`.

**Review by DavyCats — CHANGES_REQUESTED** [review]

File: readgroup.wdl (hunk 2 above)
Comment (ffinfo) [reply readgroup.wdl:41]: In this case all that happens on readgroup here is QC

**Review by DavyCats — APPROVED** [review]

**Pull request merged**